Filtration/ViewModels/ItemFilterCommentBlockViewModel.cs

```diff
@@ -47,7 +47,17 @@ public override void Initialise(IItemFilterBlockBase itemfilterBlock, IItemFilte
 
         public IItemFilterCommentBlock ItemFilterCommentBlock { get; private set; }
 
-        public string Comment => ItemFilterCommentBlock.Comment;
+        public string Comment
+        {
+            get
+            {
+                return ItemFilterCommentBlock.Comment;
+            }
+            set
+            {
+                ItemFilterCommentBlock.Comment = value;

```

Comment:
This line should be followed by `RaisePropertyChanged();` - this won't make a difference to the current code but it ensures that if the Comment property is ever changed from the ViewModel in future updates that the UI will update at the same time.